Files/Views/SettingsPages/About.xaml

```diff
@@ -176,6 +181,21 @@
                     </local:SettingsBlockControl.ExpandableContent>
                 </local:SettingsBlockControl>
 
+                <local:SettingsBlockControl
+                    x:Uid="SettingsPrivacy"
+                    Title="Privacy policy"
+                    Width="{StaticResource SettingsBlockControlDefaultWidth}"
+                    HorizontalAlignment="Left"
+                    Loaded="SettingsBlockControl_Loaded">
```

Comment:
Should we include the actual text in the app so that it works offline and can also be trasnlated?